### ISSUE TITLE
Reminder keeps coming up after dismissing

### ISSUE THREAD

**Issue by jethrogb** [Ericsson/exchangecalendar#546]
I often have this problem where after I dismiss a reminder it comes back (in a seemingly endless loop).

My software:

Linux
EWS Provider 3.8.0
Lightning 4.7.4
Thunderbird 45.4.0

Please let me know what additional information I can provide.

**Comment by tyldum** [comment]
Is it recurring tasks? If so, try turning off reminders for past events under global calendar settings.

**Comment by jethrogb** [comment]
Do you mean Preferences > Calendar > Reminders > When a Reminder is Due: > Show missed reminders? I have turned that off now, I'll see if it makes a difference.

**Comment by tyldum** [comment]
Yup. Was on a non-English tbird at the time, so hard to describe. Glad you found it.

**Comment by t2d** [comment]
Thank you for explaining the work-around. The constant reminders made my Thunderbird unusable.
But in general, I want to see missed reminders. Is there a solution to only this plugin?

**Comment by vicmunoz** [comment]
I'm affected too. Workaround worked fine for me. Thanks
My config:
Linux
EWS Provider 3.9.0
Lightning 4.7.4
Thunderbird 45.7.0

**Comment by retorquere** [comment]
@jethrogb I can't find that preference but in `about:config` I changed `calendar.alarms.showmissed` to `false` assuming it does the same.

**Comment by FelixBuehler** [comment]
@jethrogb Couldn't find the setting either, so thanks for the hint.

**Comment by navid-zamani** [comment]
There’s a much worse problem here:

If you accept those missed alarms, instead of just plain closing the window with (x), **you recurring events get reset!**

How to reproduce:
1. Install and run a `radicale` or similar CalDAV server. (E.g. on your PC.)
2. Install `aCalendar` or a similar app on Android. Use `DAVx⁵` to connect to the CalDAV server.
3. Connect `Thunderbird/Lightning` to the same CalDAV server.
4. Create a recurring event (e.g. every Saturday for the whole day) with a reminder (e.g. 3 hours in advance, or even multiple reminders) on your phone.
5. Sync your phone to CalDAV (e.g. in `DAVx⁵`), then sync to `Thunderbird/Lightning`. Close `Thunderbird`(!).
6. Now wait until one event is passed and accept the reminder on your phone.
7. Sync your phone again.
8. After a while, edit one past occurrence of the event, and say you only want to edit that one, and not any future ones or all of them. Move it to another (earlier?) day. (E.g. to Friday.)
9. Sync your phone again.
10. Reopen `Thunderbird/Lightning`.
11. Get an alarm for that old occurrence because it was saved with the original reminder.
12. Accept the reminder, so TB doesn’t show it anymore.

Expected result:
Aside from ignoring the missed alarm, the moved older event should stay moved.

Actual result:
The edited older event is either moved back(!) or another (duplicate) event is created where it used(!) to be.

After long non-usage of TB, **I clicked to accept all past alarms, and in one big swoop my entire calendar of the past months was in chaos!** Which is baaad, because I use it as a journal, to see what was done and when.
(That’s why my workaround, and generally a good idea, was to turn the `radicale` file storage (which I chose for that reason) into a `git` repository, and use `kdeconnect`’s command function, to get a button on my phone that lets me commit changes to `git`. Which I do twice a day. That way I can always go back. A la `timeline`.)
